api/views.py

```diff
@@ -92,7 +92,7 @@ class CollectionList(generics.ListCreateAPIView):
     permission_classes = (drf_permissions.IsAuthenticatedOrReadOnly,)
 
     def get_queryset(self):
-        queryset = Collection.objects.all()
+        queryset = Collection.objects.all().order_by('-date_created')
```

Comment:
I'm thinking it might be good to sort by num of items in the collection, and by most recently added to as well.

Reply:
Should that be part of this PR or would it be good to make a separate one for that?